shell/shared/grain.js

```diff
@@ -1644,9 +1644,8 @@ Router.map(function () {
     path: "/apps/:appId",
     template: "appDetails",
     waitOn: function () {
-      return [
-        Meteor.subscribe("appIndex", this.params.appId),
-      ];
+      return globalSubs.concat(
```

Comment:
Hmm no I guess not, since you're not doing Meteor.subscribe() again, you're just passing in the existing subscription objects.